src/backend/storeManagers/gog/games.ts

```diff
@@ -481,10 +481,18 @@ export async function launch(
         ? wineExec.replaceAll("'", '')
         : wineExec
 
+    const wineFlagsObj = {
+      proton: ['--no-wine', '--wrapper', `'${wineBin}' run`],
+      wine: ['--wine', wineBin],
+      toolkit: [
+        '--wrapper',
+        `${wineBin} ${gameSettings.winePrefix}`,
+        '--no-wine'
+      ]
+    }
+
     wineFlag.push(
-      ...(wineType === 'proton'
-        ? ['--no-wine', '--wrapper', `'${wineBin}' run`]
-        : ['--wine', wineBin])
+      ...(wineFlagsObj[wineType as keyof typeof wineFlagsObj] || [])
```

Comment:
maybe this is a good opportunity to extract this that is repeated in both stores into a helper function? then it could be handled as a switch/case to not need to do this type stuff

something like a `wineFlagsForWine(wineType)` and that thing returns the arrays? I think it's going to be more readable and more maintainable